src/App.vue

```diff
@@ -26,18 +26,17 @@ export default {
     HeaderMain,
     NavbarFilter,
     FooterMain,
-    Card,
+    Scroll,
     InfiniteLoading
   }
 }
 </script>
 
 <style lang="scss">
-.header-top {
-  background-image: url("https://www.passionclub.ro/upload/slider/slider-1.jpg");
-  background-position: center;
-  background-repeat: no-repeat;
-  background-size: cover;
-}
-
+  .header-top {
+    background-image: url("https://www.passionclub.ro/upload/slider/slider-1.jpg");
```

Comment:
download the image and put it in your assets folder. What if someone removes the image from the internet? Your app will not work properly